Return a default config if one is not found, for testing purposes

From: Dennis Afanasev <dennis.afa@gmail.com>


---
 src/InfUdDriver.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/InfUdDriver.cc b/src/InfUdDriver.cc
index b03c60a3..eb73d94d 100644
--- a/src/InfUdDriver.cc
+++ b/src/InfUdDriver.cc
@@ -311,7 +311,7 @@ InfUdDriver::readDriverConfigFile()
             LOG(ERROR, "Ignored bad driver configuration: '%s'", sl.c_str());
         }
     }
-    return ServiceLocator("infud:");
+    return ServiceLocator("infud:hca=mlx4_0,port=2,eth=enp8s0d1");
 }
 
 /**
